tests/item/item_constructors.cpp

```diff
@@ -124,6 +125,15 @@ class TEST_NAME : public util::test_base {
                 check_equality(successAcc, current_check::copy_constructor,
                                copied, item);
 
+                if constexpr (numDims == 1) {
+                  // Check deafult Dimension and offset
```

Comment:
```suggestion
                  // Check default Dimension and offset
```
Use a spell checker?

Reply:
Excuse me, I will work on my spelling